packages/genesys-spark-components/src/i18n/time-zone/identifiers.ts

```diff
@@ -458,3 +458,603 @@ export const timeZoneIdentifiers = [
   'UTC',
   'WET'
 ] as const;
+
+// from what public API returns from /api/v2/timezones
+export const genesysSupportedTimeZones = [
```

Comment:
Why not use `genesysSupportedTimeZones` to create a timezone type? Is there a benefit to using `string`?